.github/workflows/write-prerelease-comment.yml

```
@@ -63,38 +63,46 @@ jobs:
         with:
           number: ${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER}}
           message: |
-            A wrangler prerelease is available for testing. You can install this latest build in your project with:
+            🧪 Prerelease packages have been built and are available for testing. 🧪
+
+            <details>
+
+            ### `wrangler`
+
+            You can install this PR's `wrangler` prerelease in your project with:
 
             ```sh
             npm install --save-dev https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-wrangler-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
             ```
 
-            You can reference the automatically updated head of this PR with:
+            Or you can use `npx` to run it directly:
 
             ```sh
-            npm install --save-dev https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/prs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-wrangler-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
+            npx https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-wrangler-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }} dev path/to/script.js
             ```
 
-            Or you can use `npx` with this latest build directly:
+            ### `create-cloudflare` (C3)
 
+            You can run this PR's `create-cloudflare` prerelease in your project with:
             ```sh
-            npx https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-wrangler-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }} dev path/to/script.js
+            npx https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-create-cloudflare-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
```

Is this correct? The wrangler and pages-shared URL's have `-package-` whereas this does not

```suggestion
            npx https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-create-cloudflare-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
```

---

thanks a lot for spotting this, yeah sorry not sure how I manage to miss it here 😓👍